skll/utilities/compute_eval_from_predictions.py

```diff
@@ -10,25 +10,85 @@
 import argparse
 import csv
 import logging
+import random
 
 from skll.data import Reader, safe_float
 from skll.metrics import use_score_func
 from skll.version import __version__
 
 
+def get_prediction_from_probabilities(classes, probs, prediction_method):
+    """
+    Convert a list of class-probabilities into a class prediction. This function assumes
+    that, if the prediction method is 'expected_value', the class labels are integers.
+
+    Parameters
+    ----------
+    classes: list
+        List of str or int class names.
+
+    probs: list of float
+        Probabilities for respective classes.
+
+    prediction_method: str
+        Indicates how to get a single class prediction from the probabilities. Currently
+        supported options are  "highest", which selects the class with the highest
+        probability, and "expected_value", which calculates an expected value over
```

Comment:
What if two classes are tied for the highest probability? I believe in that case `max` returns the first one it encounters. This should be mentioned in the docstring.

Reply:
Maybe it is better to always make a random choice when two or more classes tie.